examples/redirects/cms/src/collections/Pages.ts

```diff
@@ -0,0 +1,50 @@
+import { CollectionConfig } from 'payload/types';
+import { Content } from '../blocks/Content';
+import { hero } from '../fields/hero';
+import { slugField } from '../fields/slug';
+
+export const Pages: CollectionConfig = {
+  slug: 'pages',
+  admin: {
+    useAsTitle: 'title',
+    defaultColumns: ['title', 'slug', 'updatedAt'],
+  },
+  versions: {
+    drafts: true,
+  },
+  access: {
+    read: () => true,
+  },
+  fields: [
+    {
+      name: 'title',
+      type: 'text',
+      required: true,
+    },
+    {
```

Comment:
I don't think we need `hero` or `layout` in this demo. Title and a single `richText` field for content is plenty.